content/docs/azure/deploy/install-kubeflow.md

```diff
@@ -106,7 +106,7 @@ kfctl generate all -V
 kfctl apply k8s -V
 ```
 
-    * ${KFAPP} - the name of a directory where you want Kubeflow configurations to be stored. This directory is created when you runkfctl init. If you want a custom deployment name, specify that name here. The value of this variable becomes the name of your deployment. The value of this variable cannot be greater than 25 characters. It must contain just the directory name, not the full path to the directory. The content of this directory is described in the next section.
+* ${KFAPP} - the name of a directory where you want Kubeflow configurations to be stored. This directory is created when you runkfctl init. If you want a custom deployment name, specify that name here. The value of this variable becomes the name of your deployment. The value of this variable cannot be greater than 25 characters. It must contain just the directory name, not the full path to the directory. The content of this directory is described in the next section.
```

Comment:
The numbering is incorrect in the preview:
https://deploy-preview-816--competent-brattain-de2d6d.netlify.com/docs/azure/deploy/install-kubeflow/